EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFUnpackerOutput.cc

```diff
@@ -63,9 +68,17 @@ namespace l1t
 					
 	  RegionalMuonCand muCand;
 	  RegionalMuonRawDigiTranslator::fillRegionalMuonCand(muCand, raw_first, raw_secnd, processor, tftype::bmtf);
-	  muCand.setLink(48 + processor);	//the link corresponds to the uGMT input
 
-	  LogDebug("L1T") << "Pt = " << muCand.hwPt() << " eta: " << muCand.hwEta() << " phi: " << muCand.hwPhi();
+	  if (isKalman) {
+	    muCand.setLink(48 + processor); 
```

Comment:
`setLink` is the same in the if block and the else block. It could be moved outside.